src/popup/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -128,7 +128,7 @@ export default {
     Tab,
   },
   props: {
-    id: { type: String, required: true },
+    id: { type: String, default: null },
```

Comment:
As the null is default in almost every case I suggest to do it like this:
```
id: String,
```

Reply:
It's mandatory to set default value for props with type other than `Boolean`